.changeset/fuzzy-tips-clap.md

```diff
@@ -0,0 +1,15 @@
+---
+"@medusajs/modules-sdk": patch
+"@medusajs/types": patch
+"@medusajs/authetication": patch
+"@medusajs/cart": patch
+"@medusajs/customer": patch
+"@medusajs/payment": patch
```

Comment:
```suggestion
```

Same with these as well, but gotta check with the module owners.

Reply:
good catch... will remove them